omnibridge/model_entities/models_io/file_to_text_converter.py

```diff
@@ -0,0 +1,48 @@
+import os
+from typing import Union
+
+from omnibridge.model_entities.models_io.base_model_io import TextualIO
+
+
+class FileInputHandler:
+    @staticmethod
+    def convert_file(file_path: str) -> Union[TextualIO, None]:
+        """
+        Reads the content of a file given its file path.
+
+        Args:
+            file_path (str): The path to the file.
+
+        Returns:
+            TextualIO: The content of the file ready to be consumed by models.
+        """
+
+        allowed_extensions = {
+            '.txt': FileInputHandler.convert_txt_file,
+            '.py': FileInputHandler.convert_txt_file
+        }
+
+        _, file_extension = os.path.splitext(file_path)
+
+        if file_extension.lower() not in allowed_extensions.keys():
+            raise ValueError("Error: The provided file is not a .txt file.")
```

Comment:
Just a small suggestion, let's make the error message more helpful by listing all the allowed extensions in it.

Reply:
overlooked, nice catch!